internal/juju/applications.go

```diff
@@ -614,6 +647,7 @@ func (c applicationsClient) ReadApplication(input *ReadApplicationInput) (*ReadA
 		Config:      conf,
 		Constraints: appConstraints,
 		Principal:   appInfo.Principal,
+    Placement: placement,
```

Comment:
uber nit: probably needs a pass with `gofmt` :)

Reply:
I ran gofmt and made the changes it recommended. Thanks for the feedback!